doc/source/train/user_guide.rst

```diff
@@ -224,31 +231,40 @@ To customize the ``backend`` setup, you can replace the string argument with a
 
     .. code-block:: python
 
-        from ray.train import Trainer
-        from ray.train.torch import TorchConfig
+        from ray.train.torch import TorchTrainer, TorchConfig
 
-        trainer = Trainer(backend=TorchConfig(...), num_workers=2)
+        trainer = TorchTrainer(
+            train_func,
+            torch_backend=TorchConfig(...),
+            scaling_config=dict(num_workers=2),
```

Comment:
cc @sumanthratna we'll need to be aware of updating this to use the `ScalingConfig` dataclass